ibm/provider.go

```diff
@@ -169,6 +169,7 @@ func Provider() terraform.ResourceProvider {
 			"ibm_certificate_manager_certificate":   dataIBMCertificateManagerCertificate(),
 			"ibm_cis":                               dataSourceIBMCISInstance(),
 			"ibm_cis_dns_records":                   dataSourceIBMCISDNSRecord(),
+			"ibm_cis_healthchecks":                  dataSourceIBMCISHealthCheck(),
```

Comment:
it is advisable to have `dataSourceIBMCISHealthChecks`

Reply:
done